styles/themes/base/base.variables.scss

```diff
@@ -51,6 +51,18 @@
 
   --ld-password-rule-text-color: var(--ld-main-color);
 
+  --ld-drop-down-suggestion-padding-inline: 1rem; 
+  --ld-drop-down-suggestion-padding-block: .5rem; 
+  --ld-drop-down-suggestion-border-radius: var(--ld-border-radius);
+  --ld-drop-down-suggestion-bg: var(--ld-main-color-contrast);
+  --ld-drop-down-suggestion-bg-hover: #e5e5e5; // tw neutral 200
```

Comment:
You why don't use TW variables? Like `var(--tw-color-neutral-200, "#e5e5e5")`?

This way it will use a tw variable if available and fallback to the hex color)